docs/source/notes/get_start_xpu.rst

```diff
@@ -33,13 +52,13 @@ Installation
 Binaries
 ^^^^^^^^
 
-Now that we have `Intel GPU Driver <https://www.intel.com/content/www/us/en/developer/articles/tool/pytorch-prerequisites-for-intel-gpu/2-6.html#driver-installation>`_ installed, use the following commands to install ``pytorch``, ``torchvision``, ``torchaudio`` on Linux.
+Now that we have `Intel GPU Driver <https://www.intel.com/content/www/us/en/developer/articles/tool/pytorch-prerequisites-for-intel-gpu.html#driver-installation>`_ installed, use the following commands to install ``pytorch``, ``torchvision``, ``torchaudio`` on Linux.
 
-For release wheels
+For preview wheels
```

Comment:
Hi @ZhaoqiongZ and @guangyey we want to change it back to preview ?

Reply:
Hi @atalman , nope. We will fix it within 4 hrs.